package.json

```diff
@@ -22,8 +22,10 @@
     "test": "jest --passWithNoTests --no-cache --runInBand",
     "test:watch": "npm test -- --watch",
     "test:e2e": "node_modules/.bin/cypress open",
-    "postinstall": "npx husky install"
-  },  
+    "postinstall": "npx husky install",
+    "format": "prettier --check --no-error-on-unmatched-pattern --ignore-path .gitignore . --ext ts,tsx,js,jsx,json,md",
```

Comment:
we have the pre-push hook on .husky folder. Don't you think it would be good to put the format and format fix commands there to run BEFORE the npm test? Then, when someone do a new push, it would verify the formats, fix the wrongs and then run the tests

Reply:
Yeah sure. Sounds good.